baseapp-email-templates/tests/conftest.py

```diff
@@ -0,0 +1 @@
+from tests.fixtures import *  # noqa
```

Comment:
import fixtures from baseapp_core here so you don't need to import them in your test files

```suggestion
from baseapp_core.test.fixtures import *  # noqa
```